src/style.css

```diff
@@ -1,3 +1,81 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+  padding: 0 50px;
```

Comment:
Don't set paddings to `body` - it is better to style some internal containers instead (in your case it would be `header`